### ISSUE TITLE
Could you please explain if you create one for each day and how it shows up in the date table?

### ISSUE THREAD

**Issue by OPEN1004MIND** [zoreet/habit-tracker#5]
I only read the readme, so I don't know how it works. 
What do you mean by creating a directory for each hobby and then creating an empty file? 
Could you please explain if you create one for each day and how it shows up in the date table?

It seems like creating notes by date in a directory by hobby would create too many files, is this correct?

**Comment by zoreet** [comment]
Hi @OPEN1004MIND 

Thanks for trying out Habit Tracker 21 and for leaving this feedback. What I meant ( and will look to update the readme soon ) is that you create only one file per habit. The name of the file is how the habit is presented in the table. You don't need to do anything else. You definitely don't need to create a file per date. Leave the file empty and the plugin will take care of putting all your entries in it.

I hope this clarifies it a bit. Will definitely review the README sometime next week #xmas

**Comment by OPEN1004MIND** [comment]
I understand how it works now.

My problem is that the title letter of the displayed results is too small and I want to adjust it, but how?
I've tried adjusting the font size in the plugin's CSS, but it doesn't make a difference.

**Comment by zoreet** [comment]
Did you get it to work? Did my explanation above make sense?

**Comment by OPEN1004MIND** [comment]
> Did you get it to work? Did my explanation above make sense?
Yes I see how it works now, I realized I had to create an empty file to get it to show up in the hobby tracker.
Thank you

**Comment by zoreet** [comment]
Amazing!

I'd really appreciate it if you could have a look over the new README and tell me if the tweaks I've made it any clearer. And pls share any feedback you have, would love to know what your experience is.
